index.html

```diff
@@ -45,6 +45,19 @@ <h1 id="title">
         This is a work on progress.
       </p>
     </section>
+    <section id="introduction">
+      <h2>Introduction
+      </h2>
+      <p>
+        This section is non-normative.
+      </p>
+      <p>
+        The scope of this document is to specify the behavior of the 3D content as part of the
+        [[HTML]] document. 3D content downloaded or obtained by some other mechanism
+        (such as Quick Look) is considered detached from the [[HTML]] document is out of the
+        scope of this specification.
```

Comment:
"detached" has certain meaning (e.g., detached node) so might be confusing... we might need some other concept here. 

```suggestion
        is out of the scope of this specification.
```

Reply:
I guess it's more about things being presented outside the scope of a document, right?